templates/terraform/resource.erb

```diff
@@ -131,9 +131,6 @@ func resource<%= resource_name -%><%= prop.name.camelize(:upper) -%>SetStyleDiff
 <% end -%>
 
 func resource<%= resource_name -%>Create(d *schema.ResourceData, meta interface{}) error {
-<%  if object.custom_code.custom_create -%>
```

Comment:
I think I said to remove this, but it looks already committed. Never mind in that case- can you revert the changes to this file?

Reply:
These changes still need to be removed.